kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ConvertUtils.java

```diff
@@ -352,29 +322,34 @@ public static NodeInstanceList convertToNodeInstanceList(Collection<NodeInstance
     }
 
     public static VariableInstance convertToVariable(VariableDesc variableDesc) {
-        VariableInstance instance = VariableInstance.builder()
+        return convertToVariable(variableDesc, null);
+    }
+
+    public static VariableInstance convertToVariable(VariableDesc variableDesc, String containerId) {
+        if (containerId != null && !variableDesc.getDeploymentId().equals(containerId)) {
+            throw new ProcessInstanceNotFoundException("process instance " + variableDesc.getProcessInstanceId() + " does not belong to container {}" + containerId);
```

Comment:
placeholder {} is not needed here
```suggestion
            throw new ProcessInstanceNotFoundException("process instance " + variableDesc.getProcessInstanceId() + " does not belong to container " + containerId);
```

Reply:
done